setup.py

```diff
@@ -4,6 +4,7 @@
 
 requirements = [
     'createrepo_c~=0.13',
+    'productmd',
```

Comment:
Oh I see you're using the library here. Perhaps we are already using everything we can from it?